staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -321,12 +319,21 @@ func (o *DrainCmdOptions) RunDrain() error {
 
 func (o *DrainCmdOptions) deleteOrEvictPodsSimple(nodeInfo *resource.Info) error {
 	list, errs := o.drainer.GetPodsForDeletion(nodeInfo.Name)
+	if len(list.Pods()) > 0 && o.drainer.DryRun {
+		fmt.Fprintf(o.Out, "namespace/pod (dry run)\n")
```

Comment:
Remove this line entirely, you'll not gonna get it in regular drain execution path, so it can't be here either.

Reply:
got it, removed